File: program/find-surface-area-of-a-cube/FindSurfaceAreaOfACube.cs
using System;

class FindSurfaceAreaOfACube
{
  static void Main()
  {
    Console.Write("Enter Side: ");
    decimal Side = Convert.ToDecimal(Console.ReadLine());
    decimal Area = 6 * (Side * Side);
    Console.WriteLine("Surface Area of Cube is:" + Area);
  }
}

// input:  Enter Side : 2
// output: Surface Area of Cube is: 24

File: program/find-surface-area-of-a-cube/FindSurfaceAreaOfACube.cpp
#include<iostream>
using namespace std;
int surfaceAreaOfCube(int &side){
	return 6 * side * side;
}
int main(){
	int side=0;
	cout<<"Enter Side of Cube: ";
	cin>>side;
	if(side<=0)
		cout<<"Invalid value (Side of a cube can not be Zero or less than Zero)"<<endl;
	else 
		cout<<"Surface Area : "<<surfaceAreaOfCube(side)<<endl;
}
